Apps/WebClient/src/NewClientApp/src/components/private/timeline/TimelineComponent.vue

```diff
@@ -655,6 +698,24 @@ setPageFromDate(linearDate.value);
                     >Learn more</a
                 >.
             </v-alert>
+            <v-alert
+                v-else-if="isOnlyBcCancerScreeningSelected"
+                type="info"
+                data-testid="linear-timeline-cancer-screening-disclaimer"
```

Comment:
We no longer have a non-linear timeline, so we should be removing any references to the linear timeline and not introducing new ones.